v2/config/samples/compute/v1beta20220301_virtualmachine.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: compute.azure.com/v1beta20220301
+kind: VirtualMachine
+metadata:
+  name: samplevm
+  namespace: default
+spec:
+    additionalcapabilities: null
+    applicationprofile: null
+    availabilityset: null
+    azurename: asotest-vm-ljhvpw
+    billingprofile: null
+    capacityreservation: null
+    diagnosticsprofile:
+        bootdiagnostics:
+            enabled: true
+            storageuri: null
+    evictionpolicy: null
+    extendedlocation: null
+    extensionstimebudget: null
+    hardwareprofile:
+        vmsize: Standard_A1_v2
+        vmsizeproperties: null
+    host: null
+    hostgroup: null
+    identity: null
+    licensetype: null
+    location: westus2
+    networkprofile:
+        networkapiversion: null
+        networkinterfaceconfigurations: []
+        networkinterfaces:
+            - deleteoption: null
+              primary: null
+              reference:
+                group: network.azure.com
+                kind: NetworkInterface
+                name: asotest-nic-dajxvb
```

Comment:
would be better to match this with name in `NetworkInterface` sample

Reply:
Done.